packages/eas-cli/src/commands/secrets/delete.ts

```diff
@@ -30,30 +36,51 @@ Unsure where to find the secret's ID? Run ${chalk.bold('eas secrets:list')}`;
     const projectDir = (await findProjectRootAsync()) ?? process.cwd();
     const { exp } = getConfig(projectDir, { skipSDKVersionRequirement: true });
     const projectId = await getProjectIdAsync(exp);
+    const projectFullName = await getProjectFullNameAsync(exp);
+    const projectAccountName = await getProjectAccountNameAsync(exp);
 
     if (!(await isEasEnabledForProjectAsync(projectId))) {
       warnEasUnavailable();
       process.exitCode = 1;
       return;
     }
 
-    const {
+    let {
       args: { id },
     } = this.parse(EnvironmentSecretDelete);
+    let name;
+
+    if (!id) {
+      const secrets = await EnvironmentSecretsQuery.all(projectAccountName, projectFullName);
+
+      const { secret } = await promptAsync({
+        type: 'autocomplete',
+        name: 'secret',
+        message: 'Pick the secret to be deleted:',
+        choices: secrets.map(secret => ({
+          title: `${secret.name} (${secret.scope})`,
+          value: secret,
+        })),
+      });
```

Comment:
I'd consider changing this to `autocompleteMultiselect` or having a loop over the whole code so it's possible to delete multiple secrets while running `eas secrets:delete` only once.

Reply:
I'm not sure how autocompleteMultiselect looks but with regular multiselect for deleting distribution certificates a lot of users had a problem, because they don't know that this prompt is multiselect and just hit enter without selecting anything (they assume that currently highlighted value will be deleted)

It might work if we require at leas one selected value.